instrumentation/instrumentation.gradle

```diff
@@ -70,6 +70,30 @@ subprojects { Project subProj ->
   }
 }
 
+subprojects {
+  if (!project.name.endsWith('-testing')) {
+    apply plugin: 'java'
+    apply plugin: 'maven-publish'
+    publishing {
+      publications {
+        mavenJava(MavenPublication) {
+          from components.java
+          afterEvaluate {
+            def parts = project.path.split(':')
+            if (parts.size() == 3) {
+              groupId = 'io.opentelemetry.auto.instrumentation'
+              artifactId = "instrumentation-$project.name"
```

Comment:
We may want to separate the three different types here somehow:  decorators/helpers, auto instrumentation, and manual instrumentation.

Reply:
I'm thinking let's not publish artifacts to maven central for v0.1.0 to give us more time to figure out artifact naming. For v0.1.0 we can just publish the agent jar to github release.

Reply:
I'm fine with that.  Does that mean close this PR?

Reply:
Yeah, that's probably safest